bin/reth/src/commands/node/events.rs

```diff
@@ -38,11 +37,20 @@ struct NodeState<DB> {
     current_stage: Option<CurrentStage>,
     /// The latest block reached by either pipeline or consensus engine.
     latest_block: Option<BlockNumber>,
+    /// The time of the latest block seen by the pipeline
+    latest_block_time: Option<u64>,
+    /// Hash of the head block last set by fork choice update
+    head_block_hash: B256,
+    /// Hash of the safe block last set by fork choice update
+    safe_block_hash: B256,
+    /// Hash of finalized block last set by fork choice update
+    finalized_block_hash: B256,
```

Comment:
are we okay with them being zero on launch?

Reply:
Changed to `Option<B256>`